docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
```

Comment:
q: any specific reason for requiring composer file format 3.7? The compose file seems pretty simple a first sight, and I would rather specify the lowest supported version of the format as general guideline....

Reply:
Thats a good point, I did it out of habit. I'll check to see the lowest compatible version.